app/controllers/identity.js

```diff
@@ -14,16 +14,49 @@ export default class IdentityController extends Controller {
   @tracked isCopyClicked = false;
   @tracked isVerifyClicked = false;
   @tracked profileURL = this.model.profileURL || '';
+  @tracked saveDisabled = true;
+  @tracked generateChainCodeDisabled = this.model.profileURL == undefined;
+  @tracked checkboxDisabled =
+    this.model.profileURL == undefined || this.model.Chaincode == undefined;
```

Comment:
Can we please use `===` ?
Also, is `this.model.Chaincode` working ? I think it's `this.model.chaincode`

Reply:
okay changing it